README.md

```diff
@@ -69,18 +71,29 @@ cd fossology/
 vagrant up
 ```
 
-The server must be ready at [http://localhost:8081/repo/](http://localhost:8081/repo/) and user can login the credentials using
+The server must be ready at [http://localhost:8081/repo/](http://localhost:8081/repo/) and the user can log in using the credentials
```

Comment:
The word `login` is correct in this context IMO.

Reply:
It was suggested by Grammarly (ref: https://grammarist.com/spelling/log-in-login/ )
Does the below sentence sounds good?
```
The server must be ready at [http://localhost:8081/repo/](http://localhost:8081/repo/). The login credentials are
```

Reply:
Ah ha, I learned something new today. Thanks for sharing the link.
Then grammatically both sentences are correct :+1: